Dockerfile

```diff
@@ -1,23 +1,23 @@
-FROM node:alpine
+FROM node:6-alpine
 
-ENV appdir /usr/local/app
+ENV APPDIR /usr/local/app
 
-RUN mkdir -p $appdir
-WORKDIR  $appdir
-COPY . $appdir
+RUN mkdir -p "$APPDIR"
+WORKDIR $APPDIR
+COPY . $APPDIR
 RUN npm install
 
 RUN mkdir -p /verdaccio/storage /verdaccio/conf
 ADD conf/docker.yaml /verdaccio/conf/config.yaml
```

Comment:
Right, `ADD` supports copying from a URL or archive, though I wonder who ever uses that. I'll change it.